receiver/vcenterreceiver/metadata.yaml

```diff
@@ -568,6 +576,13 @@ metrics:
       value_type: int
       aggregation_temporality: cumulative
     attributes: []
+  vcenter.vm.cpu.time:
+    enabled: false
+    description:  CPU time spent in idle, ready or idle state.
+    unit: "%"
+    gauge:
+      value_type: double 
+    attributes: [cpu_spent_state, object_name]
```

Comment:
nit: @samiura sorry to be a pain, but what do you think about `cpu_state`? For some reason `spent` sounds kind of out of place to me.

Reply:
Also since these are all performance metrics you could add the `extended_documentation: As measured over the most recent 20s interval.`

Reply:
@samiura also, would you mind posting some screenshots with the new metrics? I think I'll be ready to approve after this last batch of changes.

Reply:
No not at all, I was not happy with the naming as I mentioned. Working on it and will let you know.